snmp/datadog_checks/snmp/data/default_profiles/dell-powerconnect.yaml

```diff
@@ -0,0 +1,182 @@
+  - 1.3.6.1.4.1.674.10895.3057    # PowerConnect N3024
+  - 1.3.6.1.4.1.674.10895.3058    # PowerConnect N3048
+  - 1.3.6.1.4.1.674.10895.3059    # PowerConnect N3024P
+  - 1.3.6.1.4.1.674.10895.3060    # PowerConnect N3048P
+  - 1.3.6.1.4.1.674.10895.3061    # PowerConnect N3024F
+  - 1.3.6.1.4.1.674.10895.3063    # PowerConnect N1524
+  - 1.3.6.1.4.1.674.10895.3064    # PowerConnect N1524P
+  - 1.3.6.1.4.1.674.10895.3065    # PowerConnect N1548
+  - 1.3.6.1.4.1.674.10895.3066    # PowerConnect N1548P
+  - 1.3.6.1.4.1.674.10895.3076    # PowerConnect N3132PX
+  - 1.3.6.1.4.1.674.10895.3077    # PowerConnect N2128PX
+  - 1.3.6.1.4.1.674.10895.3078    # PowerConnect N1108T-ON
+  - 1.3.6.1.4.1.674.10895.3079    # PowerConnect N1108P-ON
+  - 1.3.6.1.4.1.674.10895.3080    # PowerConnect N1124T-ON
+  - 1.3.6.1.4.1.674.10895.3081    # PowerConnect N1124P-ON
+  - 1.3.6.1.4.1.674.10895.3082    # PowerConnect N1148T-ON
+  - 1.3.6.1.4.1.674.10895.3083    # PowerConnect N1148P-ON
+  - 1.3.6.1.4.1.674.10895.3084    # PowerConnect N3048EP-ON
+  - 1.3.6.1.4.1.674.10895.3085    # PowerConnect N3048ET-ON
+  - 1.3.6.1.4.1.674.10895.3086    # PowerConnect N3024EP-ON
+  - 1.3.6.1.4.1.674.10895.3087    # PowerConnect N3024EF-ON
+  - 1.3.6.1.4.1.674.10895.3088    # PowerConnect N3024ET-ON
+  - 1.3.6.1.4.1.674.10895.3089    # PowerConnect N1608X-ON
+  - 1.3.6.1.4.1.674.10895.3090    # PowerConnect N1608PX-ON
+  - 1.3.6.1.4.1.674.10895.3091    # PowerConnect N1616X-ON
+  - 1.3.6.1.4.1.674.10895.3092    # PowerConnect N1616PX-ON
+  - 1.3.6.1.4.1.674.10895.3093    # PowerConnect N1624X-ON
+  - 1.3.6.1.4.1.674.10895.3094    # PowerConnect N1624PX-ON
+  - 1.3.6.1.4.1.674.10895.3095    # PowerConnect N1648X-ON
+  - 1.3.6.1.4.1.674.10895.3096    # PowerConnect N1648PX-ON
+  - 1.3.6.1.4.1.674.10895.3097    # PowerConnect N2224X-ON
+  - 1.3.6.1.4.1.674.10895.3098    # PowerConnect N2224PX-ON
+  - 1.3.6.1.4.1.674.10895.3099    # PowerConnect N2248X-ON
+  - 1.3.6.1.4.1.674.10895.3100    # PowerConnect N2248PX-ON
+  - 1.3.6.1.4.1.674.10895.3104    # PowerConnect NN3208PX-ON
+  - 1.3.6.1.4.1.674.10895.3105    # PowerConnect N3224X-ON
+  - 1.3.6.1.4.1.674.10895.3106    # PowerConnect N3224PX-ON
+  - 1.3.6.1.4.1.674.10895.3107    # PowerConnect N3248X-ON
+  - 1.3.6.1.4.1.674.10895.3109    # PowerConnect N3248PXE-ON
+  - 1.3.6.1.4.1.674.10895.3110    # PowerConnect N3224T-ON
+  - 1.3.6.1.4.1.674.10895.3111    # PowerConnect N3224F-ON
+  - 1.3.6.1.4.1.674.10895.3112    # PowerConnect N3224P-ON
+  - 1.3.6.1.4.1.674.10895.3113    # PowerConnect N3248TE-ON
+  - 1.3.6.1.4.1.674.10895.3114    # PowerConnect N3248P-ON
+  - 1.3.6.1.4.1.674.10895.3115    # PowerConnect N1108EP-ON
+
+metrics:
+  - MIB: RADLAN-MIB
+    symbol:
+      name: cpu.usage
+      OID: 1.3.6.1.4.1.89.1.8.0
+#  - MIB: DNOS-SWITCHING-MIB
+#    symbol:
+#      name: cpu.usage
+#      OID: 1.3.6.1.4.1.674.10895.5000.2.6132.1.1.1.1.4.9.0
+#      extract_value: 60 Secs.*?(\d+)
+# TODO: not supported yet (?)
```

Comment:
did you find any examples for this metric ?

Reply:
Yes, [here](https://forums.cacti.net/viewtopic.php?t=47910): The string returned for instance is "5 Secs ( 41.68%) 60 Secs ( 41.14%) 300 Secs ( 41.26%)"

Reply:
Using `extract_value: 60 Secs.*?(\d+)` we should be able to extract the value from `5 Secs ( 41.68%) 60 Secs ( 41.14%) 300 Secs ( 41.26%)`.

What you can possibly do is:

1/ set in snmprec:
```
1.3.6.1.4.1.674.10895.5000.2.6132.1.1.1.1.4.9.0|4|5 Secs ( 41.68%) 60 Secs ( 41.14%) 300 Secs ( 41.26%)
```

2/ And use:
```
#  - MIB: DNOS-SWITCHING-MIB
#    symbol:
#      name: cpu.usage
#      OID: 1.3.6.1.4.1.674.10895.5000.2.6132.1.1.1.1.4.9.0
#      extract_value: 60 Secs.*?(\d+)
```

This should lead to `snmp.cpu.usage` metric with value of `41.14`

3/ Assert `snmp.cpu.usage` have value `41.14` in e2e test.

Reply:
@AlexandreYang it works, but it only takes the integer part of the number. I changed regex to `(\d+[.]?)` to match with the fractional part as well